avocado/core/remote/runner.py

```diff
@@ -209,7 +209,11 @@ def run_test(self, urls, timeout):
                                       "specified timeout (%s). Interrupting."
                                       % (timeout))
 
-        json_result = self._parse_json_response(result.stdout)
+        try:
+            json_result = self._parse_json_response(result.stdout)
+        except:
+            raise exceptions.JobError(result.stdout)
```

Comment:
Would you please include the `log_exc_info` to log the original traceback into the debug log?